uint8 FREQ_ON=255
uint8 FREQ_1HZ=1
uint8 FREQ_2HZ=2
uint8 FREQ_3HZ=3
uint8 FREQ_4HZ=4
uint8 FREQ_5HZ=5
uint8 FREQ_OFF=0

uint8 AMP_OFF=0
uint8 AMP_WEAK=64
uint8 AMP_MED=127
uint8 AMP_STRONG=190
uint8 AMP_FULL=255

uint8 INACTIVE=0
uint8 ACTIVE=1

Header header

uint8[16] active
uint8[16] freq
uint8[16] amp_max
uint8[16] amp_min
